ssi-jws/src/error.rs

```diff
@@ -1,5 +1,18 @@
 #[derive(thiserror::Error, Debug)]
 pub enum Error {
+    /// Missing curve in JWK
+    #[error("Missing curve in JWK")]
+    MissingCurve,
+    /// Curve not implemented
+    #[error("Curve not implemented: '{0}'")]
+    CurveNotImplemented(String),
+    /// Errors from p256, k256 and ed25519-dalek
+    #[cfg(feature = "k256")]
```

Comment:
should these features be `"secp256k1"` and `"secp256r1"`?

Reply:
I was asking myself the same question -- I came to the conclusion that it doesn't matter as long as we make sure to always use features and never optional dependencies as features

Reply:
never  mind I forgot about dependencies being enabled by other crates